src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -314,6 +315,33 @@ public Job call() throws SaltStackException {
     }
 
     /**
+     * Get previously run jobs.
+     * @return map containing run jobs keyed by job id.
+     * @throws SaltStackException if anything goes wrong
+     */
+    public Map<String, Job> getJobs() throws SaltStackException {
+         Result<List<Map<String, Job>>> result = connectionFactory
+                .create("/jobs", JsonParser.JOBS, config)
+                .getResult();
+         return result.getResult().get(0);
```

Comment:
If it is firmly guaranteed that there will be always one element, no matter what, then such code is OK. Otherwise nobody will like debug "Index out of range", while real error is "Server did not returned a proper data".